tools/scripts/libraries.mk

```diff
@@ -108,6 +108,13 @@ CFLAGS += -I$(AZURE_DIR)/sdk/inc
 
 endif
 
+ifneq ($(if $(findstring lwip, $(LIBRARIES)), 1),)
+include $(NO-OS)/tools/scripts/lwip.mk
+INCS += $(NO-OS)/libraries/lwip/arch
+INCS += $(NO-OS)/libraries/lwip/configs/lwipcfg.h
```

Comment:
as i said, this will need to be configurable in the future, but leave it as-is for now